docs/source/quick_start.rst

```diff
@@ -0,0 +1,140 @@
+
+        .. tab:: Debian/Ubuntu
+
+            .. code::
+
+                apt install rustc cargo build-essential libgtk-4-dev git
+
+        .. tab:: Fedora
+
+            .. code::
+
+                yum install rust cargo make gcc gcc-c++ gtk4-devel pango-devel git
+
+        Note that Packetry requires a minimum Rust version of 1.74. If your distribution's packages are older than this, use `rustup <https://rustup.rs/>`__ to get the latest Rust toolchain and manage your Rust installation.
+
+        **Install Packetry**
+
+        Clone the Packetry repository:
+
+        .. code::
+
+            git clone https://github.com/greatscottgadgets/packetry.git
+
+        **Build Packetry**
+
+        .. code::
+
+            cd packetry
+            cargo build
+
+        Note: Do not build with ``--all-features``. All the optional features currently in the package are for debug/test purposes only, and will prevent or degrade normal use of the application if enabled. See ``Cargo.toml`` for details.
+
+.. tab:: macOS
+
+    .. tab:: Binary
+
+        Homebrew installation coming soon!
+
+    .. tab:: Cargo
+
+        **Install Prerequisites**
+
+        Use `rustup <https://rustup.rs/>`__ to install the latest Rust toolchain and manage your Rust installation.
+
+        Use `Homebrew <https://brew.sh/>`__ to install GTK 4:
+
+        .. code::
+
+            brew install gtk4
+
+        **Install Packetry**
+
+        Clone the Packetry repository:
+
+        .. code::
+
+            git clone https://github.com/greatscottgadgets/packetry.git
+
+        **Build Packetry**
+
+        .. code::
+
+            cd packetry
+            cargo build
+
+        Note: Do not build with ``--all-features``. All the optional features currently in the package are for debug/test purposes only, and will prevent or degrade normal use of the application if enabled. See ``Cargo.toml`` for details.
+
+.. tab:: Windows
+
+   Download and run the Windows installer from the `latest release <https://github.com/greatscottgadgets/packetry/releases/latest>`__.
+
+
+Connect Cynthion
+----------------
+
+Connect Cynthion's CONTROL port to your analysis host computer running Packetry. Connect TARGET C to your target host computer. If your target host and analysis host are the same computer, please be aware of `this issue <https://github.com/greatscottgadgets/packetry/issues/122>`__.
+
+You can connect Cynthion's TARGET A port to your target device at this time, but you may wish to delay this connection until after Packetry is capturing. This will ensure that Packetry can observe the entire device enumeration process.
+
+.. image:: ../images/cynthion-connections-packetry.svg
+  :width: 800
+  :alt: Connection diagram for using Cynthion with Packetry.
+
+
+Launch Packetry
+---------------
+
+.. tab:: Binary
+
+   Run the Packetry executable.
+
+.. tab:: Cargo
+
+   In the ``packetry`` directory run:
+
+    .. code::
+
+        cargo run
```

Comment:
It's probably worth mentioning here that they can use `cargo install --path .` to build and install the binary to their `.cargo/bin` directory, which will mean running `packetry` will work from anywhere if that's in their `PATH`. 

Or, after running `cargo build --release`, they can copy `target/release/packetry[.exe]` to somewhere else in their `PATH`, like `/usr/local/bin` on Unix-like systems.

Reply:
`cargo install --path .` seems to me like it *should* be the best option for folks who have used `rustup`, but it fails in an unfriendly way for folks who haven't. #sigh#

How about this (with the hint)?
```
cargo build --release
sudo cp target/releases/packetry /usr/local/bin
```
Hint: You can copy the `packetry` executable to any directory in your `PATH` instead of `/usr/local/bin`. If you've used `rustup` and have `.cargo/bin` in your `PATH`, `.cargo/bin` is a great choice!